src/components/FontStager/index.jsx

```diff
@@ -2,6 +2,10 @@ import { Component } from 'react';
 import FoutStager from 'react-fout-stager';
 import './index.css';
 
+/**
+ * Responsible for loading the application typefaces progressively
+ * using FOUT stage techniques.
+ */
 export default class FontStager extends Component {
```

Comment:
Could we have a dark colour background with white/gray text for code segments?

<img width="754" alt="screen shot 2018-04-16 at 11 09 14 am" src="https://user-images.githubusercontent.com/3766511/38819175-af144f40-4168-11e8-84c1-7589c28e1115.png">

Reply:
<img width="629" alt="screen shot 2018-04-16 at 4 15 32 pm" src="https://user-images.githubusercontent.com/285899/38835674-6e13cd8a-4191-11e8-976e-cc2860acd775.png">